packages/mobile/__mocks__/react-native-firebase.ts

```diff
@@ -1 +1,30 @@
-export default {}
+let toFail = false
+
+function links() {
+  return {
+    createShortDynamicLink: jest.fn(async () => {
+      if (toFail) {
+        throw new Error('test error')
+      }
```

Comment:
nit: we're using private state to test a failure case. A slightly nicer approach would be expose a `createShortDynamicLink` mock that doesn't change and let the test itself set the kind of return value it wants for it.

Reply:
This sound like a good idea, will change it